watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/dye_desalination.py

```diff
@@ -131,6 +202,39 @@ def set_operating_conditions(m):
     dye_sep.P1.eta_pump.fix(0.75)  # pump efficiency [-]
     dye_sep.P1.lift_height.unfix()
 
+    if hasattr(m.fs, "gac"):
+        m.fs.tb_nf_gac.properties_out[0].temperature.fix(298.15)
```

Comment:
@hunterbarber The inlet T and P to the translator block are not fixed since the ZO property package (the inlet property package) does not support T or P. So we fix the temperature and pressure outlet of this translator, which is then fed to the GAC inlet. Let me know if that doesn't address your question or if I'm misunderstanding something.

Reply:
Oh it is the outlet to the translator, I saw `...gac.properties_out...` and mistaken it for the outlet of the GAC block, my fault.